client/app/common/authentication/authentication.js

```diff
@@ -1,10 +1,13 @@
 import Authentication from "./authentication.service";
 import uiRouter from "angular-ui-router";
+import { hookProvider } from "./hook.js";
```

Comment:
@nicksellen I did it like this, because my test needs the createHook function from this file. It gets a bit messy, if you have default and regular export: 
`import defaultExport,{ theThingYouWant } from "someModule"`
I think, i would like to change the test to not be dependent on createHook, but i didn't see an easy approach.